packages/core-ui/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "watch": "webpack -d --watch",
     "dev:build": "webpack -d",
-    "build": "yarn run sass:build && yarn run copy && webpack -p --config webpack.production.config.js",
+    "build": "yarn run copy && webpack -p --config webpack.production.config.js",
```

Comment:
i guess the copy task should now be part of the dev:theme target, or refactor to remove the need to copy ?